docs/list_of_tutorials.jl

```diff
@@ -19,6 +19,7 @@ if generate_tutorials
     skip_execute = [
         "heldsuarez.jl",                 # broken
         "risingbubble.jl",               # broken
+        "densitycurrent.jl",             # broken
```

Comment:
It would be really great if this tutorial were actually executable.

Reply:
> It would be really great if this tutorial were actually executable.

@charleskawczynski in what sense? Aren't they executed?

Reply:
The intention is that these tutorials are executed in the docs. Adding this line skips executing this tutorial.

Currently, this tutorial doesn’t result in valid markdown to be executed. I suggest first adding this tutorial to the list of tutorials pages array in `docs/list_of_tutorials.jl`, and look at the preview of that the tutorial looks like in the documentation.

You’ll see that comments break up function blocks in the resulting markdown (in the `generated` directory in the docs). If you then inspect the doc build, you’ll see that nearly all of these blocks fail to execute.

Proper comments (see an existing tutorial that does properly execute, or #1181 which has many such fixes) need to be added to prevent breaking up function blocks.

Reply:
@charleskawczynski #1181 has conflicts specifically in `docs/list_of_tutorials.jl` Is this the actual PR you meant to point me to?

Reply:
Yes, many of the fixes I'm referring to are in the docs, not changes in `docs/list_of_tutorials.jl` in #1181.